src/core/tests/type_prop/einsum.cpp

```diff
@@ -372,14 +372,272 @@ TEST_F(TypePropEinsumTest, incorrect_equation_not_broadcastable_shapes) {
                     HasSubstr("Input dimensions labeled with ellipsis for Einsum must be broadcastable."));
 }
 
-TEST_F(TypePropEinsumTest, incorrect_equation_missed_ellipsis) {
+TEST_F(TypePropEinsumTest, missed_out_ellipsis) {
     const std::string equation = "a...b,b...->a";
 
-    const auto input_shapes = Shapes{{11, 1, 4, 3}, {3, 11, 7, 5}};
+    const auto input_shapes = Shapes{{11, 1, 4, 3}, {3, 11, 7, 4}};
+    const auto inputs = make_inputs(element::f32, input_shapes);
+    const auto o = make_op(inputs, equation);
+    EXPECT_EQ(o->get_equation(), equation);
+    EXPECT_EQ(o->get_element_type(), element::f32);
+    EXPECT_EQ(o->get_output_size(), exp_einsum_outputs_count);
+    EXPECT_EQ(o->get_output_partial_shape(0), PartialShape({11}));
+    EXPECT_THAT(get_shape_symbols(o->get_output_partial_shape(0)), Each(nullptr));
```

Comment:
why this is not symbols[0]?

Reply:
Test was using static shapes without any symbols, switched to PartialShape + added check for symbols where applicable